pyproject.toml

```diff
@@ -1,3 +1,10 @@
 [tool.black]
 skip-string-normalization = true
-line-length = 120
+line-length = 100
+
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
```

Comment:
This is not aligned with the [template repository](https://github.com/canonical/template-operator). Do you think you could open a PR in there with the changes and discuss them if needed?

Reply:
I have updated it to be the same as the template project